tools/cell_census_builder/validate.py

```diff
@@ -74,58 +78,85 @@ def validate_all_soma_objects_exist(soma_path: str, experiment_builders: List[Ex
         assert datetime.fromisoformat(census.metadata["created_on"])
         assert "git_commit_sha" in census.metadata
 
-        for name in [CENSUS_INFO_NAME, CENSUS_DATA_NAME]:
-            assert soma.Collection.exists(census[name].uri)
-            assert census[name].soma_type == "SOMACollection"
+        with tiledb.Group(census.uri) as census_group:
```

Comment:
Using `tiledb` directly is fine for now, but it would be better if we used the SOMA API for the `is_relative` test.  This points out a gap in in the SOMA API!

Please:  a) file a feature request for the SOMA equivalent of `is_relative`,    b) add a TODO/reminder in this code to port to that new API when available.

Reply:
ditto for the same code below.

Reply:
do I file it in the [TileDB-SOMA](https://github.com/single-cell-data/TileDB-SOMA) repo?

Reply:
Yep, that would be perfect. It is a case where we just forgot to add the ability to introspect this property.  Nice dogfooding find!!!

Reply:
https://github.com/single-cell-data/TileDB-SOMA/issues/999